actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -291,7 +298,7 @@ private void updateServerNodes()
 
     private Task<NodeAddress> locateActiveActor(final RemoteReference<?> actorReference)
     {
-        final NodeAddress address = localAddressCache.get(actorReference);
+        final NodeAddress address = localAddressCache.asMap().get(actorReference);
```

Comment:
My understanding is we would want to use `localAddressCache.getIfPresent()` instead of reaching in and getting the Map from the Cache. There is three places where we do the `asMap()` call.

https://github.com/ben-manes/caffeine/blob/master/caffeine/src/main/java/com/github/benmanes/caffeine/cache/Cache.java#L54